contracts/Governance/VTreasuryV8.sol

```diff
@@ -0,0 +1,85 @@
+pragma solidity 0.8.20;
+
+import { SafeERC20, IERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @title VTreasuryV8
+ * @author Venus
+ * @notice Protocol treasury that holds tokens owned by Venus
+ */
+contract VTreasuryV8 is Ownable {
+    using SafeERC20 for IERC20;
+
+    // WithdrawTreasuryToken Event
+    event WithdrawTreasuryToken(address tokenAddress, uint256 withdrawAmount, address withdrawAddress);
+
+    // WithdrawTreasuryNative Event
+    event WithdrawTreasuryNative(uint256 withdrawAmount, address withdrawAddress);
```

Comment:
We could index the addresses of the events

Reply:
Added here [29b65b8](https://github.com/VenusProtocol/venus-protocol/pull/345/commits/29b65b8e0767eeef1f45af9c292be6b750c31282)